payroll/migrations/0029_auto_20200205_1722.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.2.9 on 2020-02-05 17:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('payroll', '0028_removeuserzipcode'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='salary',
+            name='extra_pay',
+            field=models.DecimalField(decimal_places=2, max_digits=10, null=True),
+        ),
+        migrations.AlterField(
+            model_name='salary',
+            name='amount',
+            field=models.DecimalField(decimal_places=2, max_digits=10, null=True),
+        ),
+        migrations.AddConstraint(
+            model_name='salary',
+            constraint=models.CheckConstraint(check=models.Q(models.Q(models.Q(_negated=True, amount=0), ('amount__isnull', False)), models.Q(models.Q(_negated=True, extra_pay=0), ('extra_pay__isnull', False)), _connector='OR'), name='total_pay_not_null'),
```

Comment:
Looks like this line [is causing the linter to fail](https://travis-ci.com/datamade/bga-payroll/builds/147595219#L844). You can add `# noqa` (with two spaces before the hash) to the end of lines to omit them from linting. Here's [an example](https://github.com/datamade/bga-payroll/blob/53e08b88f511d101586c2bc90e0029af250f9fc5/bga_database/settings.py#L14)!

Reply:
I created a new migration file to replace this one. I'm not getting any linting errors locally with this new file, so I'm going to push it as it is now. If it's still a problem, let me know!

Reply:
Looks like the linter [is still upset](https://travis-ci.com/datamade/bga-payroll/builds/147621817#L844). :-(